python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -420,6 +420,7 @@ def predict(self,
                 data: Union['SparkXShards', 'SparkDataFrame'],
                 batch_size: int=32,
                 feature_cols: Optional[List[str]]=None,
+                output_cols: Optional[List[str]]=None,
```

Comment:
So in this case, can we support the previous single output column to specify the column name, i.e. instead of "prediction "(default), user can specify the output_col

Reply:
Seems it's not supported currently? @leonardozcm

Reply:
For now our output_col logic is like:
1. If single output, user **does not and should not** pass output_col to.predict()
2. if multi-output, user should pass a list of output_col to predict(), and the number of output_cols is exactly corresponding to the predict columns.

Reply:
For multi-output, output_cols is always required right? where do you add this assert?

Reply:
If so, the interface would be better if single output can also specify the output_col

Reply:
yes it's need to add an assert, and we plan to add a func (in next pr) to allocate output_cols automatically in case users don't specify them.